### ISSUE TITLE
grafana rpm is not AGPLv3

### ISSUE THREAD

**Issue by mnagaya** [grafana/grafana#39908]
```
rpm -qip https://dl.grafana.com/oss/release/grafana-8.1.5-1.x86_64.rpm | grep License
warning: https://dl.grafana.com/oss/release/grafana-8.1.5-1.x86_64.rpm: Header V4 RSA/SHA256 Signature, key ID 24098cb6: NOKEY
License     : "Apache 2.0"
```

**Environment**:
- Grafana version: 8.1.5
- Data source type & version: ?
- OS Grafana is installed on: Red Hat, CentOS, RHEL, and Fedora
- User OS & Browser: ?
- Grafana plugins: ?
- Others: ?

**Comment by mnagaya** [comment]
The fedora pakaging guideline is here:
https://fedoraproject.org/wiki/Packaging:LicensingGuidelines#Valid_License_Short_Names
https://fedoraproject.org/wiki/Licensing:Main#Software_License_List

**Comment by marefr** [comment]
Fixed since Grafana v8.2.6

```
❯ rpm -qip https://dl.grafana.com/oss/release/grafana-8.2.6-1.x86_64.rpm | grep License
warning: https://dl.grafana.com/oss/release/grafana-8.2.6-1.x86_64.rpm: Header V4 RSA/SHA256 Signature, key ID 24098cb6: NOKEY
License     : AGPLv3
```
